python/ert/__init__.py

```diff
@@ -1,3 +1,11 @@
+import warnings
```

Comment:
I think this file will be overwritten with ert/__init__.py from libres, will that have any unintended side-effects?